docker-compose.yaml

```diff
@@ -0,0 +1,185 @@
+services:
+  probo:
+    image: "ghcr.io/getprobo/probo:latest"
+    environment:
+      # Required secrets (use secure values in production)
+      PROBOD_ENCRYPTION_KEY: "thisisnotasecretAAAAAAAAAAAAAAAAAAAAAAAAAAA="
```

Comment:
@SachaProbo we must make the application crash instead of default value in probod, it will be more secure and avoid people running with default value.

@kermitsxb could you in case the key does not exist generate one? Or it's better to keep just default one and ask people to set it properly?